rotkehlchen/tests/api/test_assets.py

```diff
@@ -193,7 +208,11 @@ def test_ignored_assets_endpoint_errors(rotkehlchen_api_server_with_exchanges, m
         status_code=HTTPStatus.BAD_REQUEST,
     )
     # Check that assets did not get modified
-    assert set(rotki.data.db.get_ignored_assets()) == set(ignored_assets)
+    expected_tokens = set(
+        ignored_assets +
+        [EthereumToken('0x824a50dF33AC1B41Afc52f4194E2e8356C17C3aC')],
```

Comment:
Which asset is this and why is it not defined as a constant somewhere?

Reply:
Is the kick token. Can define it as constant

Reply:
do that please.

Reply:
> do that please.

only inside the file though